config.yml

```diff
@@ -1,2 +1,2 @@
 ---
-statements_graph: https://raw.githubusercontent.com/rightsstatements/data-model/master/rights-statements.ttl
\ No newline at end of file
+statements_graph: https://raw.githubusercontent.com/rightsstatements/data-model/feature/2015-07-15-update/rights-statements.ttl
```

Comment:
I'm reluctant to change the pointer, here, since the branch will be deleted when that PR is merged.

Let's remove this change and just merge https://github.com/rightsstatements/data-model/pull/11 ?